include/netlink/route/link/inet6.h

```diff
@@ -36,6 +36,16 @@ extern int		rtnl_link_inet6_get_addr_gen_mode(struct rtnl_link *,
 extern int		rtnl_link_inet6_set_addr_gen_mode(struct rtnl_link *,
 							  uint8_t);
 
+extern int		rtnl_link_inet6_get_flags(struct rtnl_link *,
+							  unsigned int *);
```

Comment:
the implementation uses type `uint32_t`. It must be consistent, and `uint32_t` seems better as it's closer to the actual value. Same for the setter.